### ISSUE TITLE
README Usage section isn't correct

### ISSUE THREAD

**Issue by BMorearty** [ko1/allocation_tracer#10]
After the first revision of the README, several changes were made to the output of the first code sample without making corresponding updates to the explanatory text. This makes it very hard to tell what the output means. For example:

```
will show

{["test.rb", 6]=>[50000, 0, 47440, 0, 1, 0],
 ["test.rb", 7]=>[50000, 4, 47452, 0, 6, 0],
 ["test.rb", 8]=>[50000, 7, 47456, 0, 6, 0]}
```

...is followed by text that references `test.rb:6` with numbers 5; 44,290; 0; and 6. Of these, 0 is the only number actually shown on that line of output.

Also, each line includes 5 metrics, but the beginning of the "Allocation tracing" section only mentions four metrics: count, total_age, max_age, and min_age.

I would submit a PR to fix it but I don't understand it.

**Comment by meetme2meat** [comment]
@BMorearty  Same on this one as well.

> {:T_OBJECT=>[3434, 96563, 0, 0, 1, 0, 0, 2],
>  :T_STRING=>[3435, 96556, 2, 1, 1, 1, 1, 1, 2]}

This output means that the age of 3434 T_OBJECT objects are 0, 96563 objects are 1 and 2 objects are 7. Also the age of 3435 T_STRING objects are 0, 96556 objects are 1 and so on.

Note that these numbers includes living objects and dead objects. For dead objects, age means lifetime. For living objects, age means current age.

**Comment by ko1** [comment]
sorry i missed this ticket. @meetme2meat is correct!

**Comment by meetme2meat** [comment]
@ko1 Not an issue I have worked on README.md a bit want to look at this. https://github.com/ko1/allocation_tracer/pull/11

**Comment by BMorearty** [comment]
@meetme2meat Thank you for the fix in #11. Closing this issue.
